.github/workflows/continuous-integration.yml

```diff
@@ -32,7 +32,7 @@ jobs:
         uses: "actions/checkout@v2.0.0"
 
       - name: "Install PHP with extensions"
-        uses: "shivammathur/setup-php@1.7.3"
+        uses: "shivammathur/setup-php@1.7.4"
```

Comment:
you should use the `@v1` tag. shivammathur takes care of re-tagging the `@v1` to whatever new release he publishes

see https://github.com/shivammathur/setup-php/releases/tag/v1

Reply:
I have noticed, yes, but to be honest, I'm not sure this is the best idea (retagging).

I think it would be great if GitHub came around to properly resolving versions!

Also see https://github.community/t5/GitHub-Actions/Referencing-major-version-of-GitHub-action-only-fails-to/m-p/40562#M4280.

Reply:
ohh my. seems they did reinvent the wheel instead of learning from package managers doing this kind of stuff for years :-/